samples/extauthz/src/main.go

```diff
@@ -46,9 +47,10 @@ const (
 )
 
 var (
-	httpPort = flag.String("http", "8000", "HTTP server port")
-	grpcPort = flag.String("grpc", "9000", "gRPC server port")
-	denyBody = fmt.Sprintf("denied by ext_authz for not found header `%s: %s` in the request", checkHeader, allowedValue)
+	serviceaccount = flag.String("allowserviceaccount", "a", "allow service account expected from source principle")
```

Comment:
1. rename the variable `serviceaccount` to `serviceAccount`
2. change the flag to `allow_service_account`
3. fix the typo in the usage, expected -> extracted

Reply:
Thanks a lot.
For mannual test, rebuild pilot image with `make docker.pilot` is enough for testing. right?

Reply:
this has nothing to do with pilot